app/jenkins-for-jira-ui/package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "jenkins-for-jira-ui",
   "version": "0.1.0",
+	"engines" : {
```

Comment:
Locked in the node version. Otherwise, if you're using a version outside of these versions, installing packages fails.